ts/sagas/wallet.ts

```diff
@@ -778,6 +794,19 @@ function* completionHandler(
 ) {
   // -> it should proceed with the required operations
   // and terminate with the "new payment" screen
+
+  /**
+   * ask and check PIN before proceed with payment
+   * TODO: apply suggestion - https://github.com/teamdigitale/italia-app/pull/446
```

Comment:
what is the meaning of this TODO? the link points to this same PR

Reply:
@valentinacala any pointers about this?

Reply:
at now it had been consolidated. The comment will be deleted with the last commit